package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@licenserocks/kit",
-  "version": "3.4.2",
+  "version": "3.4.3",
```

```suggestion
  "version": "3.4.6",
```
Please bump version and merge it @adiusz